message_sender/serializers.py

```diff
@@ -22,6 +22,36 @@ class Meta:
             'helper_metadata', 'created_at', 'updated_at')
 
 
+class JunebugInboundSerializer(serializers.HyperlinkedModelSerializer):
+    """
+    Maps fields from Junebug onto fields expected by the Inbound model.
+    """
+    reply_to = serializers.CharField(source='in_reply_to')
+    to = serializers.CharField(source='to_addr')
+    channel_id = serializers.CharField(source='transport_name')
+    channel_data = serializers.JSONField(source='helper_metadata')
```

Comment:
If it exists and isn't `None`, then it will be in the Junebug `channel_data`. So if it exists it will be there, but there's not a guarantee that there will always be something there.